portfolio/src/main/webapp/script.js

```diff
@@ -18,11 +18,15 @@
 function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const hobbies = 
+      ['Running', 'Hiking', 'Rock Climbing', 'Photography', 'Traveling'];
+  const links =
+      ['LinkedIn', 'Github', 'Projects', 'Hobbies'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random link.
```

Comment:
bonus points if you can make the link clickable :)

hint: there is an innerHtml attribute you can use in place of innerText